src/jit/emit.cpp

```diff
@@ -1984,8 +1980,11 @@ void emitter::emitEndFnEpilog()
 {
     emitEndPrologEpilog();
 
+#ifdef JIT32_GCENCODER
```

Comment:
Sure. I wasn't sure whether or not we used the JIT32 GC encoder for ARM32, so I elected not to change this earlier.